parsec/core/fs/remote_loader.py

```diff
@@ -417,20 +426,50 @@ async def load_manifest(
             FSDeviceNotFoundError
             FSInvalidTrustchainError
         """
-        if timestamp is not None and version is not None:
-            raise FSError(
-                f"Supplied both version {version} and timestamp `{timestamp}` for manifest "
-                f"`{entry_id}`"
-            )
+        assert (
+            timestamp is None or version is None
+        ), "Either timestamp or version argument should be provided"
+        # Get the current and requested workspace entry
+        # They're usually the same, except when loading from a workspace while it's in maintenance
+        current_workspace_entry = self.get_workspace_entry()
+        workspace_entry = current_workspace_entry if workspace_entry is None else workspace_entry
```

Comment:
workspace_entry is an object or `None`, why don't use the simpler `or` construct instead ?
```python
workspace_entry = workspace_entry or current_workspace_entry
```

Reply:
I personally find it more explicit :)